modules/util/string_test.go

```diff
@@ -45,3 +45,11 @@ func TestToSnakeCase(t *testing.T) {
 		assert.Equal(t, expected, ToSnakeCase(input))
 	}
 }
+
+func TestToLF(t *testing.T) {
+	assert.Equal(t, "\na\nbc\n\n", ToLF("\r\na\r\nb\rc\n\n"))
```

Comment:
```suggestion
	assert.Equal(t, "\r\na\nbc\n\n", ToLF("\r\r\na\r\nb\rc\n\n"))
```

Reply:
Or how do we want to handle this case?
I can live both with the current approach of removing it completely, and my proposed approach of only converting `\r\n -> \n`